guides/k8-akka-cluster/src/main/paradox/index.md

```diff
@@ -0,0 +1,600 @@
+
+The application name will also be referenced in the SBT or Maven build file, and as such the application name will form the Docker image name when containerized.
+
+
+## 2. Handling environment variables
+
+To establish Akka cluster within the Kubernetes container, we will be adding the following system properties when containerizing your application:
+
+```
+-Dakka.actor.provider=cluster
+-Dakka.remote.netty.tcp.hostname="$AKKA_REMOTING_BIND_HOST"
+-Dakka.remote.netty.tcp.port="$AKKA_REMOTING_BIND_PORT" -Dakka.cluster.seed-nodes.0="akka.tcp://${AKKA_ACTOR_SYSTEM_NAME}@${AKKA_SEED_NODE_HOST}:${AKKA_SEED_NODE_PORT}"
+-DactorSystemName=${AKKA_ACTOR_SYSTEM_NAME}
+```
+
+Some of the system properties values are derived from environment variables, i.e. `AKKA_SEED_NODE_HOST`. These environment variables will be supplied by the Kubernetes StatefulSet.
+
+| Environment Variable    | Description                         | Example Value |
+|-------------------------|-------------------------------------|---------------|
+| AKKA_REMOTING_BIND_HOST | The hostname of the Kubernetes Pod. | `myapp-2`     |
+| AKKA_REMOTING_BIND_PORT | The Akka remoting port.             | 2551          |
+| AKKA_SEED_NODE_HOST     | The hostname of the first container in the StatefulSet. | `myapp-0` |
+| AKKA_SEED_NODE_PORT     | The Akka remoting port of the seed node. In most cases this value should match `AKKA_REMOTING_BIND_PORT`. | 2551 |
+| AKKA_ACTOR_SYSTEM_NAME  | The name of the `ActorSystem` of your application. For the purpose of this guide, we'll match the `ActorSystem` name with the application name. | 'myapp' |
+
+
+The following code can be used by your application to set up the `ActorSystem` based on the name specified by `actorSystemName` system property:
+
+```
+val actorSystemName = sys.props.get("actorSystemName")
+  .getOrElse(throw new IllegalArgumentException("Actor system name must be defined"))
+
+val actorSystem = ActorSystem(actorSystemName)
+```
+
+
+## 3. Publishing to Docker registry
+
+Next we will containerize the application and publish its Docker image. Please proceed with either [SBT instructions](#Publishing-to-Docker-registry---SBT) or [Maven instructions](#Publishing-to-Docker-registry---Maven) accordingly.
+
+
+### 3.1 Publishing to Docker registry - SBT
+
+We will be using the [SBT Native Packager](http://www.scala-sbt.org/sbt-native-packager/) plugin to containerize the application.
+
+Enable SBT Native Packager in your project by adding the following line in the `project/plugins.sbt`:
+
+```
+addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.2.0")
+```
+
+Next, follow the steps for a [SBT multi-module project](SBT-multi-module-project) or [SBT single-module project](SBT-single-module-project).
+
+
+#### 3.1.1 SBT multi-module project
+
+Follow this section if your application is part of a multi-module SBT project.
+
```

Comment:
No, I don't mean the alternatives for single or multi project, I meant the alternatives to use JavaAppPackaging or JavaAppServer.